### ISSUE TITLE
Accessibility issue with <IdeHome> cards with external links

### ISSUE THREAD

**Issue by s100** [carbon-design-system/ibm-cloud-cognitive#2213]
## What package(s) are you using?

- [ ] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [x] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.

We make use of `<IdeHome>`, which supports a prop `sets`, which is an array of set objects, where each set has a property `tasks`. In our case, one of those tasks is:

```js
const card = {
  cardLinkType: 'external',
  cardHref: 'https://ibm.biz/ACEv11ContainerDocs',
  cardSize: 'standard',
  taskHeading: translate('homepage.cards.learnMore.title'),
  taskHeadingDescription: translate('homepage.cards.learnMore.description'),
  taskGraphic: <Education />,
  taskId: 'learn_more'
}
```

Note the `cardLinkType` of `'external'`, and the supplied `cardHref`. The card renders like this:

![image](https://user-images.githubusercontent.com/9932290/186166465-2d7434fd-646a-4fc4-9545-e12fefc2c2ea.png)

When the user clicks on the card, our hyperlink is opened in a new browser tab. We have had an accessibility issue raised explaining that the user should be warned when a link opens in a new window, but the card does not give any such warning. It does provide a little icon indicating this, but the icon is not useful for users using screen readers. My suggestion would be to add alt text to this icon saying "Opens in new window", or similar, but I will leave it with you.

> Is this issue related to a specific component?

`<IdeHome>`

> What did you expect to happen? What happened instead? What would you like to
> see changed?

See above.

> What browser are you working in?

Chrome, although this isn't a browser-specific issue

> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?

`@carbon/ibm-cloud-cognitive-cdai@1.22.12`

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

IBM App Connect Enterprise. We would like to be able to fix this issue by mid-September 2022.

## Steps to reproduce the issue

See above.

## Additional information

See above for screenshots.

**Comment by elycheea** [comment]
@s100 The CDAI does not have active maintainers at this time. We’re happy to review any PRs if you’d like to contribute these changes though.

**Comment by s100** [comment]
If `@carbon/ibm-cloud-cognitive-cdai` is not supported, does that mean that it's deprecated? If so, is it deprecated in favour of some other, actively-supported package?

**Comment by elycheea** [comment]
@s100 These components are marked as legacy in [Storybook](https://carbon-for-ibm-products.netlify.app/?path=/story/cd-ai-legacy-components). While there is the [layouts section](https://pages.github.ibm.com/cdai-design/pal/layouts/library/) on the PAL site, I’m unsure how it’s being maintained or what the plans are for it longer term.

**Comment by s100** [comment]
@elycheea We use `<IdeSlideOverPanel>`, `<IdeDataTable>`, `<IdeImporting>`, `<IdeEmptyState>` and `<IdeHome>`, all from `@carbon/ibm-cloud-cognitive-cdai`. I see from Storybook that all of these components are marked as legacy. What are we supposed to use instead? What is the intended upgrade path here?

**Comment by s100** [comment]
https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2244 was merged on 1 September 2022. Now awaiting a new release of `@carbon/ibm-cloud-cognitive-cdai` with this fix in.

**Comment by s100** [comment]
Actually I'm wrong, it looks like we have [`1.22.19` now](https://github.com/carbon-design-system/ibm-cloud-cognitive/releases/tag/%40carbon%2Fibm-cloud-cognitive-cdai%401.22.19)! However, I don't see this new release on npm. In fact, at the time of writing, the latest version of `@carbon/ibm-cloud-cognitive-cdai` on the public npm registry is 1.22.17 - it looks like 1.22.18 and 1.22.19 have not been published yet.

@elycheea what can we do to make 1.22.19 generally available?

**Comment by s100** [comment]
Fixed in `@carbon/ibm-cloud-cognitive-cdai@1.22.19`
